src/redux/state.js

```diff
@@ -1,82 +1,96 @@
-import { rerenderEntireTree } from '../render';
+const store = {
 
-const state = {
-  profilePage: {
-    posts: [
-      {
-        id: '1',
-        message: 'Hi, how are you',
-        likesCount: '12',
-      },
-      {
-        id: '2',
-        message: 'It`s my first posts',
-        likesCount: '7',
-      },
-    ],
-    newPostText: '',
+  _state: {
+    profilePage: {
+      posts: [
```

Comment:
Consider going through the entire app and moving such dummy data in a separate json file/files - to show their purpose.

Reply:
Soon I will create a server where my data will be stored

Reply:
Yes, it's a good strategy.
However, it will be easier for your to switch between the real server and your mocks if you follow my advice.

Reply:
!